File: Lei_Garay_Code/Homework01/src/com/revature/question04/Factorial.java
package com.revature.question04;

public class Factorial {
	
	static void factorial(int num)
	{
		int output = num;
		while( output > 1 ){
			num = num * (output-1);
			output--;
		}
		System.out.println(num);
	}
	
	public static void main(String[] args) {
		factorial(10);
	}
}

File: Edwin_Faican_Code/Week 3/JavaScript Assisgment/js/factorial.js
//Implement HTML input/output. 

const factorial = function(n) {
    if(n === 0 || n === 1) {
        return 1;
    }
    return fact(n, n-1);
}

const fact = function(n, m) {
    if(m === 1) {
        return n;
    }
    return n * fact(m, m-1);
}